Kryptonite/SignatureLog.swift

```diff
@@ -68,6 +68,29 @@ struct SSHSignatureLog:LogStatement {
         self.date = date
     }
     
+    func getUserAndHost() -> (user:String, host:String)? {
+        guard   let hostAuth:HostAuth = try? HostAuth(jsonString: self.hostAuth),
+                let hostname = hostAuth.hostNames.first
+        else {
+            return nil
+        }
+        
+        guard displayName.contains("rejected") == false else {
```

Comment:
can we make this detect a prefix of `"rejected: "` in case some hostname / username in the future actually contains "rejected"?

Reply:
👍